core/extract/extractor.py

```diff
@@ -207,8 +208,11 @@ def save_logs(
             connections_file.close()
 
         # Save the replacements
+        if self.config['log_location']:
```

Comment:
I think the logic here should be: if endpoint or username is given, then run get_copy_replacements, if log location is given, then copy_replacements = replacements

Reply:
The reason why I say this is because I feel like its better to rely on system tables - so its preferable imo to query from sys tables when endpoint and user are provided, instead of relying on wrangling a local file

Reply:
yes the logic works that way